<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WIM experiment control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>821</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>821</width>
       <height>197</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Connection and configuring</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_connect_it">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect interrogator</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_set_it_parameters">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>220</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set interrogator parameters</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_choose_folder_to_save">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose folder to save</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_folder_to_save">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>88</y>
        <width>391</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>none</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_connect_printer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>52</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect printer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_set_printer_parameters">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>53</y>
        <width>220</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set printer parameters</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_printer_homing">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>54</y>
        <width>220</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set printer to home</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_printer_move_bed_down">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>90</y>
        <width>220</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Move printer bed down</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>821</width>
       <height>197</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Static measurements</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_set_static_measurements_params">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>11</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Static meas parameters</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_static_measurements">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Static measurements</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="recording">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>821</width>
       <height>197</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Dynamical measurements</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_set_dynamical_measurements_params">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Dynamical meas parameters</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_dynamical_measurements">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>10</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Dynamical measurements</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>821</width>
       <height>197</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Work with saved data</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_choose_file_to_load">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose file to load</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_plot_from_file">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>53</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot results</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_file_to_load">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>none</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>731</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Inspect data</string>
      </property>
      <widget class="QPushButton" name="pushButton_save_single_line_to_csv">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Save single line to csv</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_plot_single_slice_of_static">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Plot single slice across</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_axis_to_plot_static_slice">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>30</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Y</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_coordinate_to_plot_static_slice">
       <property name="geometry">
        <rect>
         <x>171</x>
         <y>25</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>250</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_FBG_to_plot_static_slice">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>30</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>17</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>30</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>FBG</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>30</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Channel</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_channel_to_plot_static_slice">
       <property name="geometry">
        <rect>
         <x>431</x>
         <y>29</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>821</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Log</string>
    </property>
    <widget class="QTextBrowser" name="LogField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>781</width>
       <height>281</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>300</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clearLog">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear log</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_save_parameters"/>
    <addaction name="action_load_parameters"/>
    <addaction name="action_delete_all_figures"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_save_parameters">
   <property name="text">
    <string>Save parameters</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_load_parameters">
   <property name="text">
    <string>Load parameters</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_delete_all_figures">
   <property name="text">
    <string>Delete all figures</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
